www/docs/main/quickstart.mdx

```diff
@@ -70,56 +71,114 @@ export type AppRouter = typeof appRouter;
 
-For implementing tRPC endpoints and routers. Install in your server codebase.
-
-`npm install @trpc/server@next`
-
-For making typesafe API calls from your client. Install in your client codebase (`@trpc/server` is a peer dependency of `@trpc/client`).
-
-`npm install @trpc/client@next @trpc/server@next`
-
-For generating a powerful set of React hooks for querying your tRPC API. Powered by [@tanstack/react-query](https://tanstack.com/query/v4/docs/adapters/react-query).
-
-`npm install @trpc/react-query@next @tanstack/react-query`
-
-For a set of utilies for integrating tRPC with [Next.js](https://nextjs.org/).
-
-`npm install @trpc/next@next`
+tRPC is broken up into several packages so you can install only what you need. Make sure to install the packages you want in the proper sections of your codebase.
+
+<table width="100%">
+  <thead>
+    <tr>
+      <th>Purpose</th>
+      <th>Location</th>
+      <th>Install command</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Implement endpoints and routers</td>
+      <td>Server</td>
+      <td>
+        <code>npm install @trpc/server</code>
+      </td>
+    </tr>
+    <tr>
+      <td>Call procedures on client</td>
+      <td>Client</td>
+      <td>
+        <code>npm install @trpc/client @trpc/server</code>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        React hooks powered by{' '}
+        <a
+          href="https://tanstack.com/query/v4/docs/adapters/react-query"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          @tanstack/react-query
+        </a>{' '}
+      </td>
+      <td>Client</td>
+      <td>
+        <code>npm install @trpc/react-query @tanstack/react-query</code>
+      </td>
+      </tr>
+      <tr>
+      <td>
+        Next.js integration utilities </td>
+      <td>Next.js</td>
+      <td>
+        <code>npm install @trpc/next</code>
+      </td>
+    </tr>
+
+  </tbody>
+</table>
 
 ### Installation Snippets
 
-<!-- remark-plugin-npm2yarn doesn't play nice with shiki so have to do these tabs manually -->
-
-**npm**
-
-```bash
-npm install @trpc/server@next @trpc/client@next @trpc/react-query@next @trpc/next@next @tanstack/react-query
-```
-
-**yarn**
-
-```bash
-yarn add @trpc/server@next @trpc/client@next @trpc/react-query@next @trpc/next@next @tanstack/react-query
-```
-
-**pnpm**
-
-```bash
-pnpm add @trpc/server@next @trpc/client@next @trpc/react-query@next @trpc/next@next @tanstack/react-query
-```
-
-## Defining a router
-
-Let's walk through the steps of building a typesafe API with tRPC. To start, this API will only contain two endpoints:
+Here are some install scripts to add tRPC to your project. These scripts include every `@trpc/*` package so feel free to remove what you don't need!
+
+<!--
+    Shiki doesn't play nice in <Tabs /> so I had to rebuild it myself.
+    The styles are all coming off of the shiki stylesheet other than
+    "manual-nord" which can be found in /src/css/custom.css.
+ -->
+
+<Tabs>
```

Comment:
We should add these tabs to the other installation snippets too:

https://www-git-shew-docsquickstart-cleanup-trpc.vercel.app/docs/v10/nextjs#1-install-deps

https://www-git-shew-docsquickstart-cleanup-trpc.vercel.app/docs/v10/react#1-install-dependencies

https://www-git-shew-docsquickstart-cleanup-trpc.vercel.app/docs/v10/aws-lambda#1-install-deps

https://www-git-shew-docsquickstart-cleanup-trpc.vercel.app/docs/v10/express#1-install-deps

https://www-git-shew-docsquickstart-cleanup-trpc.vercel.app/docs/v10/fastify#install-dependencies

https://www-git-shew-docsquickstart-cleanup-trpc.vercel.app/docs/v10/fetch#install-dependencies

(Noticed when I grabbed these that we are not consistent with `Install deps`/`Install dependencies`)

Reply:
Good shout, we should probably do an `<InstallationSnippet/>` component or something to share. I didn't do it here.